climakitae/util/utils.py

```diff
@@ -772,90 +772,118 @@ def summary_table(data: xr.Dataset) -> pd.DataFrame:
     return df
 
 
-def convert_to_local_time(data: xr.DataArray, selections) -> xr.DataArray:
+def convert_to_local_time(
+    data: xr.DataArray | xr.Dataset,
+    grid_lon: float | None = None,
```

Comment:
Please import `UNSET` from `constants` and use it as the default value. `None` should be reserved for situations where code attempted to execute and failed for some reason.

Reply:
You'll also need to change any checks for `is None` to `is UNSET`